src/main/java/br/com/alura/cibus/controller/CozinhaController.java

```diff
@@ -0,0 +1,114 @@
+package br.com.alura.cibus.controller;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.validation.Valid;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import br.com.alura.cibus.controller.dto.CozinhaForm;
+import br.com.alura.cibus.exceptions.NotFoundException;
+import br.com.alura.cibus.modelo.Cozinha;
+import br.com.alura.cibus.repository.CozinhaRepository;
+
+@Controller
+public class CozinhaController implements ErrorController {
+
+    private CozinhaRepository cozinhaRepository;
+
+    public CozinhaController(CozinhaRepository cozinhaRepository) {
+        this.cozinhaRepository = cozinhaRepository;
+    }
+    
+    @RequestMapping("/error")
+    public String handleError() {
+    	return "/error/not-found";
+    }
+
+    @GetMapping("/cozinhas")
+    public String listar(Model model) {
+    	
+        List<Cozinha> listaCozinhas = this.cozinhaRepository.findByOrderByNome();
+        model.addAttribute("listaCozinhas", listaCozinhas);
+
+        return "/listagem";
```

Comment:
Como teremos mais cruds, seria bom a JSP estar numa pasta `cozinhas` ou algo semelhante.

Aí  ficaria:

```suggestion
        return "/cozinhas/listagem";
```